README.md

```diff
@@ -133,23 +133,59 @@ docker-compose -f docker-compose.test.yml up --build --exit-code-from test
 ~~~
 
 ### Developing / Debugging
-Build and start your container in "developer mode"
+
+1.  Build your container in "developer mode":
```

Comment:
Now this is quite long guide with extended info. I would create extra page for that and link from `README.md`. This is not essential to run component.

Reply:
OK, I'll do it.